snapcraft/internal/sources.py

```diff
@@ -310,6 +346,10 @@ def __init__(self, source, source_dir, source_tag=None,
     def provision(self, dst, clean_target=True, keep_zip=False):
         zip = os.path.join(self.source_dir, os.path.basename(self.source))
```

Comment:
please use a different name than `zip`, `zip` is a python keyword and it might get confusing.